docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -323,13 +321,17 @@ ENTRYPOINT ["/docker-entrypoint.sh"]
 
 ## Ory Tunnel
 
-The Ory Tunnel has the same functionality and behaves the same way 
-as the Proxy with these exceptions:  
+The main role of the Ory Tunnel is exposing the Ory APIs under the same top-level domain 
+as your application to ensure that there are no CORS issues.
+
+The Tunnel shares most of its code with the Ory Proxy, and from the user's point of view
+it works in a similar way. There are, however, several important differences between these tools: 
 
+  ❌ Ory Tunnel doesn't handle traffic routing
+  ❌ Ory Tunnel doesn't mirror Ory URLs and as such doesn't add the `.ory` prefix to any of the Ory URLs
```

Comment:
ah so it doesn't mirror the UI urls? or only the APIs? not sure - a bit confused by this sentence

Reply:
Not sure how to answer that, need support from someone more dev-savvy :<

Reply:
it does not mirror but tunnel the API's - there's no `.ory/` prefix that you need to add to path when calling an endpoint. 
UI is available under `localhost:4000/ui/welcome` as well you can initiate login flow by calling `localhost:4000/self-service/login/browser` :)

Reply:
But the traffic still goes through the tunnel to your app?

Reply:
I understand that it does not route the traffic through the `.ory` prefix on the URL and forwards the requests to the Ory apis on their natural paths.

But the part about `Ory tunnel doesn't handle traffic routing` sounds wrong since the tunnel does indeed handle traffic to the Ory Apis (the request goes through the tunnel to get there).

From the perspective of the App not being behind the tunnel i can understand, but then why would you need to provide your App URL to the tunnel?

Reply:
Ah okay i just reread the documentation from the cli. 
I see it can be run separate from your application. 
I think you need to reword this sentence though


❌ Ory Tunnel doesn't handle traffic routing to your application, but instead to Ory APIs.